client/src/components/Workflow/Editor/Index.vue

```diff
@@ -453,20 +499,9 @@ export default {
             node.onUnhighlight();
         },
         onLint() {
-            if (this.requiresReindex) {
-                const r = window.confirm(
-                    "Workflow steps have been added or removed since last save, the workflow needs to be saved before best practices can be analzyed. Save workflow?"
-                );
-                if (r == false) {
-                    return;
-                }
-                this.onSave(true);
-            } else {
-                this._ensureParametersSet();
-                // See notes in Lint.vue about why refresh is needed.
-                this.$refs.lint.refresh();
-                showLint();
-            }
+            this._ensureParametersSet();
+            this.onDeactivate();
+            this.showInPanel = "lint";
         },
         onUpgrade() {
             this.attemptRefactor([{ action_type: "upgrade_all_steps" }]);
```

Comment:
This function ... doesn't exist ?
```suggestion
            this.onAttemptRefactor([{ action_type: "upgrade_all_steps" }]);
```